src/test/java/ca/team2706/frc/robot/subsystems/DriveBaseTest.java

```diff
@@ -87,6 +92,12 @@ public void testControlMode() {
      */
     @Test
     public void testAbsoluteGyro() {
+
+        new Expectations() {{
+            Config.getROBOT_START_ANGLE();

```

Comment:
You should just be able to use `Config,ROBOT_START_ANGLE.get()` as a non-mocked value since it doesn't require any actual hardware.

Reply:
I did not like having to create that method Config.getROBOT_START_ANGLE() but I seemed to need it for the unit test. The problem is that the expected values used in the test assume a ROBOT_START_ANGLE of 0.0 degrees but somebody might come along and want to change the value in the Config file and they should not have to update the unit test so that it passes. Recall that ROBOT_START_ANGLE represents the heading of the robot from the horizontal at the start of the match. We figured that (at least for in our minds for now) that the robot will be positioned facing downfield at the start of the match when the robot is powered on. We therefore had to change Config.ROBOT_START_ANGLE to 90.0 and the test failed. Also, at one point, maybe it will be decided that ROBOT_START_ANGLE will be 180.0 or something else. We don't want to have to rewrite the test every time if changes. From what I saw with jMockit, we can easily mock methods that return values. Therefore, it seemed to me that I was forced to create that Config.getROBOT_START_ANGLE(). If you have a way of achieving what I need to do above, please let me know because I would prefer to get rid of that Config.getROBOT_START_ANGLE(). Perhaps one way of doing it would be to mock the FluidConstant value() method when the FluidConstant object is ROBOT_START_ANGLE. Is that possible?

Reply:
Maybe I'm not seeing something here but why wouldn't we want to have the start angle of the robot always be 0 degrees?

In any case, you should be able to use the constant in the test itself. If you say the starting angle was 75.0 degrees and you wanted to test that the absolute angle with the given set of read sensor values is 19 degrees above the starting angle, you could just do that in the `assertEquals()`.

Reply:
Ok, I see it. Yes, that should work and it will make the mocking unnecessary. Thanks.

Reply:
This has been updated to get rid of the mocking and Config.getROBOT_START_ANGLE() has been removed. Updates have been pushed to Github.